nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/header/flow-status/flow-status.component.html

```diff
@@ -127,6 +127,16 @@
         </div>
         <div class="flex">
             <search [currentProcessGroupId]="currentProcessGroupId"></search>
+            <button
+                class="flow-analysis-notification w-8 border-l flex justify-center items-center"
+                (click)="openFlowAnalysisMenu()">
+                <i
+                    class="fa fa-lightbulb-o flow-analysis-notification-icon w-[24px] h-[24px] flex justify-center items-center"
+                    [ngClass]="flowAnalysisNotificationClass"
+                    [class.rounded-xl]="
```

Comment:
Why rounded background colors? We have a very similar UX for the bulletins which is right next to the flow analysis. We should follow that pattern.

Reply:
This change is included in my latest [patch](https://github.com/apache/nifi/pull/8974#discussion_r1710346976).